grf/grf.py

```diff
@@ -467,7 +467,7 @@ def write(self, filename, clean_build=False):
             sprites.extend(self._sounds.values())
 
         t.log(f'Adding strings')
-        sprites.extend(self.strings.get_actions())
+        sprites = self.strings.get_persistent_actions() + sprites + self.strings.get_actions()
```

Comment:
Just noticed this, do DCxx strings really need to be at the start of newgrf?

Reply:
I tested and apparently, it's not needed.